samples/bluetooth/direct_test_mode/README.rst

```diff
@@ -0,0 +1,237 @@
+.. _direct_test_mode:
+
+Bluetooth: Direct Test Mode
+###########################
+
+This sample enables the Direct Test Mode functions described in `Bluetooth Core Specification`_: Version 5.0, Vol. 6, Part F.
+
+Overview
+********
+
+The sample uses Direct Test Mode to verify the operation of the radio at the physical level, testing:
+
+* Transmission power and receiver sensitivity,
+* Frequency offset and drift,
+* Modulation characteristics,
+* Packet error rate,
+* Intermodulation performance.
+
+Test procedures are defined in the document `Bluetooth Low Energy RF PHY Test Specification`_: Document number RF-PHY.TS/5.0.0.
+You can carry out conformance tests using dedicated test equipment, such as the Anritsu MT8852 or similar, with the nRF5 running the DTM application set as device under test (DUT).
```

Comment:
nRF5: any nRF5x SoC (nRF5340, etc.)

However, this sample currently supports only nRF5340. Still, I think that we don't want to be too specific in the general description so that we don't have to change it frequently when adding support for new devices.

Reply:
For now I edited this as sample, but please @kapi-no and @bjorn-tore-taraldsen let me know if this should be treated as an application.

Reply:
Let's leave it as a sample and refer to it that way (discussed this point with @bjorn-tore-taraldsen)